Significant-Gravitas/Auto-GPT
# Add Edge browser support using EdgeChromiumDriverManager

### Background
This pull request aims to add support for the Edge browser in the web_selenium.py commands. The change will allow users to leverage the Edge browser while using Selenium for web automation tasks, in addition to the existing supported browsers.

### Changes
- Imported EdgeChromiumDriverManager from the webdriver_manager.microsoft package.
- Added EdgeOptions to the available browser options.
- Updated the driver initialization logic to use the Edge driver when the configured browser is set to "edge".

### Documentation
The changes made in this pull request are documented within the `.env.template` file. 

### Test Plan
1. Set the `USE_WEB_BROWSER` variable to `edge` within the file `.env.template`.
2. Rename the file `.env.template` to `.env`
3. Run the script and provide a prompt that requires the command `browse_website` to ensure that the Edge browser is launched and web scraping tasks are performed without any issues.
4. Test the same script with other browsers (Chrome, Safari, and Firefox) to ensure that the existing functionality remains intact.

### PR Quality Checklist
